### PR TITLE
Add minimum TVL filter

### PR DESCRIPTION
**Pull request by love-points**
Video here 

https://github.com/beefyfinance/beefy-v2/assets/100975625/843bd446-ac34-4ca0-bf30-0b11bc6eaed9

### CODE REVIEW

File: src/features/data/selectors/vaults.ts [hunk 1]
```diff
@@ -388,3 +389,15 @@ export const selectChainsHasCowcentratedVaults = createSelector(
     state.entities.vaults.byChainId[chainId]?.cowcentratedVault?.byEarnedTokenAddress,
   byEarnedTokenAddress => !!byEarnedTokenAddress && Object.keys(byEarnedTokenAddress).length > 0
 );
+
+export const getMaximumVaultTvl = (state: BeefyState) => {
```
Comment (ajoslin) [inline src/features/data/selectors/vaults.ts:393]: Something wrong with this one selector, it's saying max vault tvl is 500M but there is no vault equaling 500M. Need some tips on how to fix

Comment (love-points) [inline src/features/data/selectors/vaults.ts:393]: Need some help with this selector, it's telling me the highest tvl of any vault is 500M, but no vault equals this value

Comment (ReflectiveChimp) [reply src/features/data/selectors/vaults.ts:393]: its from a misprice of (retired) `convex-crveth` vault in the api - will see about getting that fixed - you might only want to grab the underlying tvl of active vaults instead though

Comment (love-points) [reply src/features/data/selectors/vaults.ts:393]: Good idea, added that check.

Comment (love-points) [reply src/features/data/selectors/vaults.ts:393]: Good idea, added.

**Comment by love-points** [comment]
Anything else needed here or are we good to merge? 👀

**Comment by ReflectiveChimp** [comment]
> Anything else needed here or are we good to merge? 👀

Can you resolve the merge conflicts please

**Comment by love-points** [comment]
> > Anything else needed here or are we good to merge? 👀
> 
> Can you resolve the merge conflicts please

Done!

**Pull request merged**